lib/coinatmradar/coinatmradar.js

```diff
@@ -103,8 +103,14 @@ function getMachines (info) {
 }
 
 function sendRadar (data) {
+  const url = _.get(options, ['coinAtmRadar', 'url']);
```

Comment:
I'm really concerned about this. Lodash arguments are reversed. This means you didn't test this code, so if I miss this in code review we're going to spend a lot of time debugging, or it could even lead to a bug in production. We need to test all changes and added features, at least with a unit test.

The trick with lodash/fp is that the last argument is almost always the object we want to work on. In this case we want to pull a field out of options, so options would be the last argument. This is handy for fp stuff because we can do:

```
then(_.get('coinAtmRadar.url'))
```

instead of:

```
then(options => _.get('coinAtmRadar.url', options)
```

Reply:
True I didn't tested it and I'm still trying to get the difference between lodash and lodash/fp
Ok, thanks for the clarification.
I just came up with that exception once I was missing the option in my lamassu.json

Reply:
Yeah, it's really confusing, I've made those mistakes many times. That makes it even more important to test and try to use types. We should talk about this more.